### ISSUE TITLE
Read Value and Attribute

### ISSUE THREAD

**Issue by href** [spurin/xmldataset#13]
I'm trying to figure out how to read the following xml (abridged):

```python
xml = """
<document>
     <amount currency="USD">123.00</amount>
</document>
"""
```

I'm trying to get the amount and the currency into the same dataset but I am not successful. This is what I have tried:

```python
import xmldataset

profile = """
document
    amount = dataset:records
        currency = dataset:records
"""

xmldataset.parse_using_profile(xml, profile)
>>> {'records': [{'amount': '123.00'}]}
```

With the approach above the currency attribute is ignored. I tried working around this as follows:

```python
profile = """
document
    amount
        currency = dataset:records
    amount = dataset:records
"""

xmldataset.parse_using_profile(xml, profile)
>>> {'records': [{'currency': 'USD', 'amount': '123.00'}, {'currency': 'USD', 'amount': '123.00'}]}
```

Unfortunately this doesn't work either: Now I get duplicate records. In my actual code the records differ slightly (I have a more complex profile).

Is there a way get both the attribute of an element and the elements value into a dataset?

**Comment by spurin** [comment]
href : Thanks for pointing this out, looking at this briefly and testing your approach, I think you've found a bug here that will need to be addressed.  I will see about getting another release which fixes this.  I think the approach that you first took was the right approach and this is how I'll be targeting this issue.

**Comment by davidmarcano** [comment]
Any updates on this issue? I ran into the same problem.

**Comment by spurin** [comment]
@davidmarcano - thanks so much for the feedback and a possible solution, this looks great.  I will try and put some time in within the next 2 weeks to see if I can get a resolution on this one to finally put it to rest 👍

**Comment by spurin** [comment]
Fix implemented from Davidmarcano in Issue #14, as per the Issue 14, have added those involved in this thread to the contributors section (on the next release).  Appreciate your efforts.
